nixos/modules/services/misc/sssd.nix

```diff
@@ -0,0 +1,32 @@
+{ config, lib, pkgs, ... }:
+with lib;
+let
+  cfg = config.services.sssd;
+  sssdConf = pkgs.writeText "sssd/sssd.conf" cfg.sssdConf;
```

Comment:
is sssdConf useful here? I might be misreading, but it looks like it doesn't actually get installed.